docs/SUMMARY.md

```diff
@@ -8,6 +8,7 @@
 ## Creating content
 
 * [Courses](courses/README.md)
+   - [Creating your audition video](courses/audition.md)
```

Comment:
should we put this under the "premium" section since it's only relevant for premium courses? These docs are intended to be relevant for anyone creating a course using our authoring tools and the long-term vision is many of these people will create content that doesn't necessarily get featured in our premium course library

Reply:
I posted to Slack this morning that some of what's under "Premium" now appears to have been superseded by what's under courses/design. An alternative location would be right at the top to make it more findable for people who want to become instructors...

Reply:
@gvwilson Ah, ok. Maybe we just add a `[Premium]` label next to certain things in the navbar? WDYT?

Reply:
:+1:

Reply:
Should I go ahead and remove what's currently under Premium that's been replaced by the new course design section and include it in this PR?

Reply:
@gvwilson seems the (premium course) 'development process' part should be kept but moved and the 'course structure' part can be removed entirely if those guidelines are made clear elsewhere

Reply:
@gvwilson have you had a chance to react to my last comment above on Jan 15? If you're confident this PR includes the most recent info from `/create/authoring` then we should get it merged in so we can delete the original page per @mcls's request

Reply:
I just pushed another update that flattens the structure and removes the redundant description of premium course content rules - I think this one is now ready to merge.